Underanalyzer/Mock/GameContextMock.cs

```diff
@@ -28,24 +55,31 @@ public void DefineMockAsset(int assetIndex, string assetName, AssetType assetTyp
         }
         assets[assetIndex] = assetName;
     }
-    public string GetMockAsset(int assetIndex, AssetType assetType)
+    
+    /// <summary>
+    /// Fetches an asset from <see cref="MockAssets"/>.
+    /// </summary>
+    /// <param name="assetType">The asset type of the asset that should be fetched.</param>
+    /// <param name="assetIndex">The index of the asset that should be fetched.</param>
+    /// <returns></returns>
```

Comment:
Probably should either not have returns, or just say that it returns the string name of the asset, or null if not found.

Reply:
missed this